### PR TITLE
New cargo shuttle for fland

### PR DESCRIPTION
**Pull request by Mangohydra**
## About the PR
new cargo shuttle for fland

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

**Changelog**

:cl:
- tweak: New cargo shuttle for fland

### CODE REVIEW

**Comment by lzk228** [comment]
i think everywhere should be the same shuttle.

**Comment by DogZeroX** [comment]
I think everwhere should have different shuttles.

File: Resources/Prototypes/Entities/Stations/base.yml [hunk 1]
```diff
@@ -55,6 +55,15 @@
...
+        - /Maps/Shuttles/mining.yml 
+
+- type: entity
+  id: BaseStationShuttlesFland
+  abstract: true
+  components:
+    - type: GridSpawn
+      paths:
+        - /Maps/Shuttles/cargo_fland.yml
         - /Maps/Shuttles/mining.yml
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Stations/base.yml:67]: remove that part and just add it to fland prototype

**Comment by Hmeister-fake** [comment]
I like cargo shuttles being different, adds flavor to cargo and adds the possibility for stations with a different type of cargo only compatible with the specific cargo shuttle of the station (like europa, it wouuld need a different "shuttle").

**Comment by EmoGarbage404** [comment]
i defer to @Emisse on this one but generally there is no reason to add more maps to the game for such a limited purpose. Cargo shuttles are getting used by 10 players max, so you get a lot less mileage compared to an evac shuttle for not a lot of gain. Especially considering the fact that the size isn't really an issue, like how it is for evac on highpop maps.

**Comment by metalgearsloth** [comment]
ATM there's no easy way to manage all of the shuttles for a station short of copying the yaml unless I start making separate components for each thing.

**Comment by Emisse** [comment]
i dont really want to set the precedent of adding unique cargo shuttles everywhere, core i was already iffy on. each one will add more maintenance to anyone doing changes that affects map code or things affecting cargo shuttle

**Comment by UbaserB** [comment]
> i dont really want to set the precedent of adding unique cargo shuttles everywhere, core i was already iffy on. each one will add more maintenance to anyone doing changes that affects map code or things affecting cargo shuttle

they said they'd maintain it, and its a separate file to the fland map as well

**Comment by IProduceWidgets** [comment]
Another vote for the rule of cool from me. Of they're every breaking it should be easy enough to yoink them for the default shuttle as long as their dock arrangement is consistent.

**Comment by Emisse** [comment]
im sorry but this creates a lot of overhead for testing things regarding station entities, this should ideally be converted to something like a string on the map prototype itself like the evac shuttle rather than stored as a station entity

**Comment by Emisse** [comment]
also yes probably reverting cores cargo shuttle as well until this is done

**Pull request closed without merging**